agent/agent.py

```diff
@@ -106,8 +106,8 @@ def load(self, fp):
 
         else:  # Create new
             logger.info("Create new runner [%s]" % fp)
-            module_path = "hasalTask"
-            object_name = "HasalTask"
+            module_path = "iskakalunan"
+            object_name = "Iskakalunan"
```

Comment:
It changes to hard code the module path to iskkaklunan, I will suggest we can put a parameter to control this..